apps/nextjs-website/src/helpers/chatbot.helper.tsx

```diff
@@ -0,0 +1,47 @@
+'use client';
+import { useEffect, useState } from 'react';
+import { getChatbotQueries, sendChatbotQuery } from '@/lib/chatbot';
+import { ChatbotQueries } from '@/lib/chatbot/queries';
+
+export const useChatbot = () => {
+  const [isLoaded, setIsLoaded] = useState(false);
+  const [isAwaitingResponse, setIsAwaitingResponse] = useState(false);
+  const [sessionId, setSessionId] = useState<string | null>(null);
+  const [queries, setQueries] = useState<ChatbotQueries>([]);
+  useEffect(() => {
+    if (sessionId) {
+      return;
+    }
+
+    // Request sessionID form chatbotAPI
+    setSessionId('sessionID');
+  }, [sessionId]);
+
+  useEffect(() => {
+    if (!sessionId) {
+      return;
+    }
+
+    getChatbotQueries(sessionId).then((response) => setQueries(response));
+    setIsLoaded(true);
+  }, [sessionId]);
+
+  const sendQuery = (queryMessage: string) => {
+    setIsAwaitingResponse(true);
+    sendChatbotQuery({
+      sessionId: sessionId || '',
```

Comment:
Here this fallback on empty string `''` is how to proper manage sandQuery when `sessionId` is missing ? Because a i think i could be more usefull manage missing case with an error.

Reply:
Add a guard for now